multihost_test/Bugzillas/data/bz_667593_1.sh

```diff
@@ -0,0 +1,24 @@
+# $1 - user
```

Comment:
You don't need this file anymore as you can call `add_password_to_group.sh`.

Reply:
add_password_to_group takes input as :
# $1 - group
# $2 - password

but bz_667593_1.sh takes input as :

# $1 - user
# $2 - password

both have different function from each other

Reply:
bz_667593.sh is replaced by add_password_to_group.sh